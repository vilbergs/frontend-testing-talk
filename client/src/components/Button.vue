<template>
  <button class="beautiful-styling" style="background: blue">
    <slot />
  </button>
</template>
