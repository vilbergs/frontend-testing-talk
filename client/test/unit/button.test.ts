import Button from '@/components/Button.vue'
import { render } from '@testing-library/vue'
import { expect, it } from 'vitest'

it('toUpperCase', () => {
  const { container } = render(Button)

  expect(container).toMatchSnapshot()
})
