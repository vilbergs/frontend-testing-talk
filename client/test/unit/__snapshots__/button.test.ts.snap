// Vitest Snapshot v1

exports[`toUpperCase 1`] = `
<div>
  <button
    class="beautiful-styling"
    style="background: blue;"
  >
    
    
  </button>
</div>
`;
